scijava-ops-engine/src/main/java/org/scijava/ops/engine/matcher/simplify/SimplifiedOpDescriptionGenerator.java

```diff
@@ -63,12 +65,13 @@ public String simpleDescriptions(OpEnvironment env, OpRequest req) {
 		var infos = SimplificationMatchingRoutine.getInfos(env, req.getName());
 		var filtered = filterInfos(infos, req);
 		String opString = filtered.stream() //
-				.map(Infos::describe) //
```

Comment:
We should consider deleting unused methods - there are now 4 different `Infos.describe...` methods - decide whether we need/want all of them